src/arbcalls/arb.jl

```diff
@@ -194,7 +194,7 @@ arbcall"void arb_rsqrt(arb_t z, const arb_t x, slong prec)"
 arbcall"void arb_rsqrt_ui(arb_t z, ulong x, slong prec)"
 arbcall"void arb_sqrt1pm1(arb_t z, const arb_t x, slong prec)"
 arbcall"void arb_root_ui(arb_t z, const arb_t x, ulong k, slong prec)"
-arbcall"void arb_root(arb_t z, const arb_t x, ulong k, slong prec)"
```

Comment:
why these two are even here? their signatures are the same?

Reply:
The latter is deprecated but kept for backwards compatibility in Arb.